src/main/java/org/spongepowered/api/util/Tristate.java

```diff
@@ -102,4 +141,23 @@ public static Tristate fromBoolean(boolean val) {
     public boolean asBoolean() {
         return this.val;
     }
+
+    /**
+     * Returns the nullable boolean representation of this tristate,
+     * where {@link Tristate#UNDEFINED} equates to null.
+     *
+     * <p>Note: To ensure that there is a one-to-one mapping of nullable
+     * booleans to and from {@link Tristate}, this method may return null
```

Comment:
may return? I see no Optional return so state it will.

Reply:
Fixed